tests/plugins/commit-message/index.js

```diff
@@ -183,8 +205,15 @@ describe("commit-message", () => {
                     .post("/repos/test/repo-test/statuses/second-sha", req => req.state === "success")
                     .reply(201);
 
+                const labelsScope = nock("https://api.github.com")
+                    .post("/repos/test/repo-test/issues/1/labels", {
+                        labels: ["feature"]
+                    })
+                    .reply(201);
```

Comment:
Can you explain why this test doesn't use `mockLabels`? Is it just because it needs the 201 status instead of the 200 in `mockLabels`?

Reply:
Looks like a copy-paste error to me. I'll update it.